static.config.js

```diff
@@ -61,7 +76,32 @@ export default {
         component: 'src/site/pages/Home',
         getData: () => ({
           supportedAssets
-        })
+        }),
+        children: generateCombinationsFromArray(supportedAssets, 'symbol', '-').map(pair => {
+          return {
+            path: `/pairs/${pair}`,
```

Comment:
It looks like this is the only place you use `pair` and everywhere else you're using `pair.split`. Consider returning a `[from, to]` tuple from generate combinations and return `from` and `to` separately in the `getData` result so you don't have to keep calling split.